ledger/complete/wal/compactor_test.go

```diff
@@ -113,7 +113,7 @@ func Test_Compactor(t *testing.T) {
 			select {
 			case <-co.done:
 				// continue
-			case <-time.After(20 * time.Second):
+			case <-time.After(60 * time.Second):
```

Comment:
Same comment: while I know we have flaky tests, this does not seem like the place to address them /cc @gomisha

Reply:
This test was failing locally, primarily.  Occasionally, it would fail on the github instance, though.